plugin/ptrace/ptracewrappers.cpp

```diff
@@ -93,7 +93,7 @@
 // FIXME:  ARM also uses sigreturn.  however, more debugging is needed before
 // ptrace works for ARM.
 # define SIGRETURN_INST_16 -1
-#elif __aarch64__
+#elif __aarch64__ || __riscv
```

Comment:
And in addition, if there is we keep some of the conditional code, then I think it should be:
> `#elif defined(__aarch64__) || defined(__riscv)`
But I agree that we probably never noticed this before is because this code is stale.  @karya0 will provide a PR to remove it.

Reply:
@aayushi363 , This code may be stale.  But let's make it correct, anyway.  As noted above, we must change this to:
> `#elif defined(__aarch64__) || defined(__riscv)`